easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-2.0.6-GCC-10.2.0.eb

```diff
@@ -0,0 +1,32 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'IQ-TREE'
+version = '2.0.6'
+
+homepage = 'http://www.iqtree.org'
+description = "Efficient phylogenomic software by maximum likelihood "
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
```

Comment:
I'm a bit surprised that a software using Eigen suppesdly has no need for other math libraries. 
I think we want this in foss level as I think Eigen will pick up on OpenBLAS

Reply:
In fact, I think it uses MPI even?

Reply:
I read the documentation and you can use both, a OpenMP and a MPI build. The question is do we need both?
In light what I said above, I think it might be best to close this PR as a newer version is already available. 
Agreed?